packages/web-react/docs/stories/Html.tsx

```diff
@@ -20,7 +20,7 @@ interface HtmlProps {
 const Html = ({ storyId, groupId }: HtmlProps) => {
   const [isDisplayed, setIsDisplayed] = useState(false);
   const [componentSource, setComponentSource] = useState('');
-  const elementId = `story--${groupId}-${storyId}--${storyId}`;
+  const elementId = `story--${groupId}-${storyId.replaceAll('-', '')}--${storyId}`;
```

Comment:
@literat – There was problem with HTML rendering when the component name is compound (Text Field, Radio Field, etc.). If you suggest another solution, I'm listening! :ear:

Reply:
I am ok with this. Thanks.

Reply:
So, finally, I have a problem with this :-) As I saw `es2021.string` in `tsconfig.json`. We are now targeting ES2015. Yeah, a little bit old, but compatible. Please use the ES2015 syntax to replace all substrings in strings.

[es2021.string](https://stackoverflow.com/questions/1144783/how-do-i-replace-all-occurrences-of-a-string-in-javascript)

Reply:
Fixed. 🙂

Reply:
Thanks.